Taskfile.yml

```diff
@@ -108,13 +110,13 @@ tasks:
   markdown:fix:
     desc: Automatically correct linting violations in Markdown files where possible
     cmds:
-      - npx markdownlint-cli --fix "**/*.md"
+      - npx markdownlint-cli --fix --ignore node_modules "**/*.md"
 
   # Source: https://github.com/arduino/tooling-project-assets/blob/main/workflow-templates/assets/check-markdown-task/Taskfile.yml
   markdown:lint:
     desc: Check for problems in Markdown files
     cmds:
-      - npx markdownlint-cli "**/*.md"
+      - npx markdownlint-cli --ignore node_modules "**/*.md"
```

Comment:
I think it will be better to use a `.markdownlintignore` file for this:
https://github.com/igorshubovych/markdownlint-cli#ignoring-files

Reply:
What do you think if we symlink `.markdownlintignore` to `.gitignore`?

They share the same syntax, in this way we wouldn't have to mantain two different files.

Reply:
I'm fine with that.

I do usually avoid the use of symlinks in open source projects because they are not so convenient for contributors using Windows to work with (by default, the `.markdownlintignore` symlink is a file that contains the text ".gitignore" when they clone the repo). But I don't think it is too serious a concern in this case, and they would be in the same situation with the current state of the project anyway.

Reply:
Ops, I totally missed that.

It seems to me a perfectly valid reason to define `.markdownlintignore` as a separate file :+1: 

(let's try at least to not make the project even more Windows unfriendly)